src/FluentAvalonia/UI/Controls/TeachingTip/TeachingTip.cs

```diff
@@ -236,7 +236,11 @@ private void CreateLightDismissIndiatorPopup()
         var popup = new Popup
         {
             // A Popup needs contents to open, so set a child that doesn't do anything.
-            Child = new Panel(),
+            Child = new Panel
+            {
```

Comment:
Can you add this comment here just so we (I) know why this is needed:
```C#
// See GH#634, Linux & mac have issue with empty popups, also Avalonia GH#17843
```

Reply:
Comment added.